### PR TITLE
updated readme file to add author, input validation also added on the…

### PR DESCRIPTION
**Pull request by AsumVictor**
### Added Contact form

- Added action and method attribute to the form
- Updated Readme file to add authors

### CODE REVIEW

**Review by Shisui6 — COMMENTED** [review]
Hi @AsumVictor, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🛠️

You have submitted a pair programming project without commits from your coding partners. Please make sure that all your coding partners make commits to this project

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -205,10 +205,10 @@ <h4>Skills</h4>
...
   <section id="contact-section">
     <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
-    <form action="#" id="form">
-      <input type="text" name="fullName" id="fullName"  placeholder="Full Name" required>
+    <form action="https://formspree.io/f/meqwdago" method="post" id="form">
+      <input type="text" name="fullName" id="fullName"  placeholder="Full Name" maxlength="30" required>
       <input type="email" name="email" id="email" placeholder="Email Address" required>
-      <textarea name="message" id="message" cols="30" rows="10"></textarea>
+      <textarea name="message" id="message" cols="30" rows="10" maxlength="500"></textarea>
       <button type="submit">Get in touch</button>
```
Comment (TadesseDev) [inline index.html:212]: - [ ] Dear @AsumVictor, good job making the form with a successful submission, But remember that all input fields are required to have the correct associated placeholder text. Please add a placeholder for your text area. 

https://github.com/AsumVictor/Porfolio/assets/69077061/c9aae4d0-07f6-417b-a93d-c2b1b4069f06

- [ ] Besides that, All input fields are required to have the `required` attribute to activate the validation on submit. Please make the `textarea` a required field.

Comment (TadesseDev) [inline index.html:212]: - [ ] Dear @AsumVictor, I notice you have applied the hover effect for your button which is great. But we need the `active` effect also. Remember for a better design we need to follow the button interactions such as hover, active.

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi Team

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7




## Highlights
 - [x] Good job overall following the template design ✅
 - [x]  Correct GitHub Flow  ✅
 - [x]  Good PR title/description  ✅
 - [x]  All linters passed  ✅
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (AsumVictor) [reply index.html:212]: Thank you very much

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Team

Your project is complete! Great job for adding the changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- form submits well
- design followed
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**